src/03_modules.py

```diff
@@ -10,22 +10,27 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+print(sys.argv)
```

Comment:
So what this problem wants you to do is print each argument from the command line individually on each line. Like if you put in the command line "python3 03_modules.py hello dude" then ideally it would print like this:

03_modules.py
hello
dude

What you have is printing all the arguments on the same line in an array like so:

['03_modules', 'hello', 'dude']

I would suggest using a for loop to get it to print correctly!